pkg/debug/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # has a fast path for stack unwinding. This also happens
 # to be a perfect place to put any other kind of debug info
 # into the package: see abuild/etc/abuild.conf.
-ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:6.7.0
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:2d71f1ab574355541e0aacad147d6d44b931f0a3
```

Comment:
Is there a way to define this in a common place which all the pkg/*/Dockerfile can include?

Reply:
We had templated Dockerfile.in before where we can put something like ALPINE_TAG here, but decided to get rid of them (it was discussion here for example https://github.com/lf-edge/eve/pull/2353). So now it is not easy. Actually I also tired to rebase all commits of the PR on top of updated eve-alpine, but it helps us to have clear Dockerfiles which we can use without any templated magic around.
Strictly speaking support for build args are not upstreamed into linuxkit and we use them only for risc now with hacked publish workflow. We use build-args there to force build for risc (we force risc arch in base image so the whole image builded for risc arch as well). But with updated linuxkit we work on in the separate PR we can remove build arg here and just use simple FROM.

Reply:
OK